Source/Particles/Collision/BinaryCollision/BinaryCollisionUtils.cpp

```diff
@@ -63,6 +63,33 @@ namespace BinaryCollisionUtils{
                 "Product species of linear Breit-Wheeler collisions must be of type electron and positron");
             return CollisionType::LinearBreitWheeler;
         }
+        else if (type == "linear_compton") {
+            amrex::Vector<std::string> species_name;
+            // Check that the first incoming species is a photon and the second is an electron/positron
```

Comment:
We should also check that the product lepton is of the same type as the incident lepton.
(i.e., if the incident lepton is an electron, then the product lepton should be an electron)